source/client.c

```diff
@@ -289,10 +311,13 @@ struct aws_mqtt_client_connection *aws_mqtt_client_connection_new(struct aws_mqt
 
     assert(client);
 
+    AWS_LOGF_DEBUG(AWS_LS_MQTT_CLIENT, "Creating new connection");
```

Comment:
Probably worth adding some contextual info here, like endpoint/port at least?

Also, most of these need to have the id=%p so that multiple connections can be differentiated

Reply:
Will add id, but endpoint:port doesn't come until connect()